kolibri/plugins/learn/assets/src/views/content-cards/index.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+
+  <component :header="header" :subheader="subheader" :contents="contentsWithLinks" :is="container" />
```

Comment:
I think this component creates an unnecessary intermediary - as you are passing in the required container as a prop, why not just have the component that has this component nested in it make the selection between the 'grid' and 'carousel' components itself?